eng/scripts/Language-Settings.ps1

```diff
@@ -5,36 +5,82 @@ $packagePattern = "*.pom"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/java-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=java%2F&delimiter=%2F"
 
-function Get-java-PackageInfoFromRepo ($pkgPath, $serviceDirectory, $pkgName)
+function Get-java-PackageInfoFromRepo
 {
-  $projectPath = Join-Path $pkgPath "pom.xml"
-
-  if (Test-Path $projectPath)
+  [CmdletBinding()]
+  Param(
+    [Parameter(Position = 0)]
+    [string]$pkgDirectoryPath,
+    [Parameter(Position = 1)]
+    [string]$serviceDirectoryName,
```

Comment:
Lets keep the parameters as is until we make the common changes.

Reply:
If I do that wouldn't it break when I try to make them positional parameters on the `eng/common` side?